lib/inquisition/fasterer/config.rb

```diff
@@ -0,0 +1,15 @@
+require 'yaml'
+
+module Inquisition
+  module Fasterer
+    class Config < ::Fasterer::Config
+      def nil_file
```

Comment:
Strange name

Reply:
I can`t rename this because this metod call from `:Fasterer::Config`